gio/Gir.toml

```diff
@@ -1129,6 +1137,20 @@ name = "Gio.ProxyAddress"
 status = "generate"
 concurrency = "send+sync"
 
+[[object]]
+name = "Gio.Resolver"
+status = "generate"
+    [[object.function]]
+    name = "records_from_res_query"
+    ignore = true
```

Comment:
Would be good to add a little comment to these why they're ignored

Reply:
Pointed to the upstream gitlab issue